src/Winners.js

```diff
@@ -0,0 +1,32 @@
+class Winners {
+  #winners = [];
+
+  #racers = [];
+
+  #maxScore = 0;
+
+  constructor(racers) {
+    this.#racers = racers;
+    this.#findMaxScore();
+    this.#findWinners();
+  }
+
+  getWinners() {
+    return this.#winners.join(',');
+  }
+
+  #findWinners() {
+    this.#racers.forEach((racer) => {
+      if (racer.getScore() === this.#maxScore) {
+        this.#winners.push(racer.getName());
+      }
+    });
+  }
+
+  #findMaxScore() {
+    const scoreList = this.#racers.map((racer) => racer.getScore());
+    this.#maxScore = Math.max(...scoreList);
+  }
```

Comment:
메서드 명이 `find`인만큼 `maxScore`를 `return`하도록 하면 어떨까요?
```suggestion
  #findMaxScore() {
    const scoreList = this.#racers.map((racer) => racer.getScore());
    return Math.max(...scoreList);
  }
```
그리고 `constructor`에서 `#maxScore`를 `set`해줘도 좋을 것 같아요.
```js
  constructor(racers) {
    this.#racers = racers;
    this.#maxScore = this.#findMaxScore();
    this.#findWinners();
  }
```

아니면 해당 메서드 네이밍을 `setMaxScoreFromRacers` 처럼 필드를 `set`한다는 의미가 담겨있으면 좋을 것 같습니다 😁

(`this.#findWinners` 메서드도 같은 의견입니당!)